docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,211 @@
+Open MPI v5.0.x series
+======================
+
+This file contains all the NEWS updates for the Open MPI v5.0.x
+series, in reverse chronological order.
+
+Open MPI version 5.0.0rc4
+-------------------------
+:Date: 30 March 2022
+
+.. caution:: MPIR API has been removed
+
+   As was announced in summer 2017, Open MPI has removed support of
+   MPIR-based tools beginning with the release of Open MPI v5.0.0.
+
+   The new PRRTE based runtime environment supports PMIx-tools API
+   instead of the legacy MPIR API for debugging parallel jobs.
+
+   See https://github.com/openpmix/mpir-to-pmix-guide for more
+   information.
+
+.. caution:: zlib is suggested for better user experience
```

Comment:
I prefer the:
```
.. admonition:: zlib is suggested for better user experience
:class: note
```
As the presence of zlib is only a performance enhancement for scale systems (i.e. use note instead of caution).

Reply:
Done.